TaskClass.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Enum representing possible task states
+
+public enum TaskStatus
+{
+	NotStarted,
```

Comment:
Maybe New instead of NotStarted just a suggestion

Reply:
Good point, changed made to New